runtime/jcl/common/shared.c

```diff
@@ -239,7 +239,8 @@ createCPEntries(JNIEnv* env, jint helperID, jint urlCount, J9ClassPathEntry** cp
 	PORT_ACCESS_FROM_VMC((J9VMThread*)env);
 
 	Trc_JCL_com_ibm_oti_shared_createCPEntries_Entry(env, helperID, urlCount);
-	
+	Assert_JCL_true(urlCount > 0);
```

Comment:
What about other callers of this method like https://github.com/eclipse/openj9/blob/2601bd3308305958029c9678bf6493e307163563/runtime/jcl/common/shared.c#L1527-L1530 ?

Should they be updated to prevent passing 0 through as well?

Reply:
We should not reach here when urlCount is 0. The other 2 callers are guarded by these two checks on the JCL side:

https://github.com/eclipse/openj9/blob/master/jcl/src/openj9.sharedclasses/share/classes/com/ibm/oti/shared/SharedClassURLClasspathHelperImpl.java#L167

https://github.com/eclipse/openj9/blob/master/jcl/src/openj9.sharedclasses/share/classes/com/ibm/oti/shared/SharedClassURLClasspathHelperImpl.java#L121
(confirmedCount is the number of confirmed URL entries in the URL classpath, so confirmedCount should always be <=urlCount)